desktop_version/src/Script.cpp

```diff
@@ -2114,25 +1798,10 @@ void scriptclass::run(void)
 			}
 			else if (words[0] == "createlastrescued")
 			{
-				if (game.lastsaved==2)
-				{
-					r=14;
-				}
-				else if (game.lastsaved==3)
-				{
-					r=15;
-				}
-				else if (game.lastsaved==4)
-				{
-					r=13;
-				}
-				else if (game.lastsaved==5)
-				{
-					r=16;
-				}
-				else
+				r = graphics.crewcolour(game.lastsaved);
+				if (r <= 1)
```

Comment:
This `r <= 1` is a bit weird. I presume you were trying to blacklist Violet's color (20) along with Viridian's? But it doesn't do that, because 20 is greater than 1.

Reply:
Whoops.